shortest_paths.py

```diff
@@ -0,0 +1,198 @@
+import numpy as np
+import networkx as nx
+from map import MapLoader
+from cell import CELL
+from monte_carlo import MonteCarlo
+import os
+import json
+
+
+class ShortestPaths:
+
+    def __init__(self, map_file, json_file="maps/opt_path.json", verbose=False):
+        """
+        Class to compute shortest paths from all staring points to a target field on a given map.
+
+        Parameters
+        ----------
+        map_file : str
+            File to load the map.
+        json_file : str
+            File where results can be stored later.
+        verbose : bool
+            if there should be intermediate output.
+        """
+        self.map_file = os.path.join(os.getcwd(), map_file)
+        self.grid = MapLoader.load_map(map_file=map_file)
+        self.mc = MonteCarlo(self.grid)
+
+        self.json_file = json_file
+        self.verbose = verbose
+
+        self.velocities = [(i, j) for i in range(-2, 3) for j in range(-2, 3)]
+
+        self.start_nodes = []
+        self.target = None
+
+        self.results = {}
+
+    def store_all_paths(self, map_files):
+        """
+        Computes and stores all shortest paths for the given maps as a JSON file.
+
+        Parameters
+        ----------
+        map_files : list(str)
+            list of map files
+        """
+
+        for map_file in map_files:
+            self.map_file = os.path.join(os.getcwd(), map_file)
+            self.grid = MapLoader.load_map(map_file=map_file)
+            self.mc = MonteCarlo(self.grid)
+
+            self.calc_shortest_paths()
+
+        json_string = json.dumps(self.results)
```

Comment:
```suggestion
        json_string = json.dumps(self.results, indent=4)
```

This ensures beautiful formatting of the json file, so that it is easy to read. Please rerun the algorithm with this, to improve the json file formatting